commands/curation/curationaudit.go

```diff
@@ -364,7 +413,10 @@ func (ca *CurationAuditCommand) auditTree(tech techutils.Technology, results map
 	sort.Slice(packagesStatus, func(i, j int) bool {
 		return packagesStatus[i].ParentName < packagesStatus[j].ParentName
 	})
-	results[strings.TrimSuffix(fmt.Sprintf("%s:%s", projectName, projectVersion), ":")] = packagesStatus
+	results[strings.TrimSuffix(fmt.Sprintf("%s:%s", projectName, projectVersion), ":")] = &CurationReport{
+		packagesStatus:        packagesStatus,
+		totalNumberOfPackages: len(depTreeResult.FlatTree.Nodes) - 1,
```

Comment:
`-1` since we also include the current module name in the FlatTree, right?
If it is the reason, please add a comment.

Reply:
sorry, its true, will leave a comment